packages/components/src/Tabs/Tabs.tsx

```diff
@@ -9,19 +9,14 @@ import { Tab } from './Tab';
 import { TabPanel } from './TabPanel';
 import { Item } from '@react-stately/collections';
 import { TabContext } from './Context';
+import { ItemProps } from '@react-types/shared';
 
-interface TabsProps
-  extends Omit<AriaTabListProps<object>, 'orientation' | 'isDisabled'>,
-    GapSpaceProp {
-  size?: 'small' | 'medium' | 'large';
-  disabled?: boolean;
-  variant?: string;
-}
-export const Tabs = ({
+export const Tabs: Tabs = ({
   space = 2,
   size = 'medium',
```

Comment:
for what is this used? @OsamaAbdellateef

Reply:
This is for padding and i think it is useless 
`      variants: {
        size: {
          small: 'px-1 pb-1',
          medium: 'px-2 pb-2 text-lg',
          large: 'px-4 pb-4 text-2xl',
        },
      },`